Lombiq.Tests.UI.Samples/Tests/DataBaseSnapshotTests.cs

```diff
@@ -0,0 +1,42 @@
+using Lombiq.Tests.UI.Attributes;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Samples.Constants;
+using Lombiq.Tests.UI.Services;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Lombiq.Tests.UI.Samples.Tests;
+
+// We can execute test on already existing databases. To test this, we will take a snapshot of the running application,
+// and use that.
+public class DataBaseSnapshotTests : UITestBase
+{
+    public DataBaseSnapshotTests(ITestOutputHelper testOutputHelper)
```

Comment:
Don't forget to rename in `MonkeyTests` and the other instances here too.

```suggestion
    public DatabaseSnapshotTests(ITestOutputHelper testOutputHelper)
```